src/java.base/share/classes/java/lang/Float16.java

```diff
@@ -282,11 +283,11 @@ public static Float16 valueOf(int value) {
     * @param  value a {@code long} value.
     */
     public static Float16 valueOf(long value) {
-        if (value < -65_504L) {
+        if (value <= -65_520L) {  // -(Float16.MAX_VALUE + Float16.ulp(Float16.MAX_VALUE) / 2)
```

Comment:
I assume you'll sync in / merge this change from the branch.

Reply:
I think this is a remnant due to my previous error to associate the PR to branch `lworld` before moving it to `lworld+fp16`.